Operators/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,49 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args) {
+
+        int number1 = 1;
+        int number2 = 2;
+        int number3 = 3;
+        int number4 = 4;
+
+        //2. Сумує їх попарно.
+        int intSum1 = number1 + number2;
+        System.out.println(number1 +" + " + number2 + " = " + intSum1);
+        int intSum2 = number3 + number4;
+        System.out.println(number3 +" + " + number4 + " = " + intSum2);
+
+        //3.Виводить результат порівняння (true якщо перша сума менша).
+        boolean compareVal1 = intSum1 < intSum2;
+        System.out.println("Compare Result 1: " + compareVal1);
+
+        //4. Збільшує першу суму на 1.
+        int intSum3 = intSum1;
+        int postIncrement = intSum3 ++;
```

Comment:
It is not necessary to create new variable for changed values but it's not a mistake. Also `postIncrement` variable is absolutely redundant here, you can just do `intSum3 ++;`

Reply:
Thanks for the comment!